### PR TITLE
ZK-4782: vflex on goldenpanel doesn't work as expected

### PR DESCRIPTION
**Pull request by DevChu**

### CODE REVIEW

File: zktest/src/archive/test2/config.properties [hunk 1]
```diff
@@ -2957,6 +2957,7 @@ B90-ZK-4431.zul=A,E,Multislider
 ##zats##B96-ZK-4835.zul=A,E,touch,mouse,drag
 ##zats##B96-ZK-4835-2.zul=A,E,touch,mouse,drag
 ##zats##B96-ZK-4817.zul=A,E,Listbox,Listgroup,Grid,group,detail,open,mobile
+##zats##B96-ZK-4782.zul=A,E,Listbox,Listgroup,Grid,group,detail,open,mobile
```
Comment (scribetw) [inline zktest/src/archive/test2/config.properties:2960]: tag

File: zktest/src/archive/test2/B96-ZK-4782.zul [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	Description:
+		
+	History:
+		Wed Apr 7 11:50:21 CST 2021, Created by jameschu
+
+Copyright (C) 2021 Potix Corporation. All Rights Reserved.
+
+-->
+<zk>
+	<goldenlayout id="gl" vflex="1" hflex="1">
```
Comment (scribetw) [inline zktest/src/archive/test2/B96-ZK-4782.zul:16]: instruction

**Review by scribetw — CHANGES_REQUESTED** [review]

**Review by scribetw — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**